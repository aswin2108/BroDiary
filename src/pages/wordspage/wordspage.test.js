import renderer from 'react-test-renderer';
import WordsPage from './wordspage.component';

it('Words page rendered correctly', ()=>{
    const wordsPage=renderer.create(
            <WordsPage/>
        ).toJSON();
      expect(wordsPage).toMatchSnapshot();
});