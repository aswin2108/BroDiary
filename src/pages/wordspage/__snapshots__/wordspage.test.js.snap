// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Words page rendered correctly 1`] = `
<div
  className="word-top"
>
  <div
    className="word-comp"
  >
    <div
      className="word"
    >
      <h1>
        Word: 
      </h1>
    </div>
    <div
      className="meaning"
    >
      <h2>
        Definition: 
      </h2>
    </div>
  </div>
</div>
`;
